web/src/main/java/org/cbioportal/web/parameter/CopyNumberCountFilter.java

```diff
@@ -0,0 +1,20 @@
+package org.cbioportal.web.parameter;
+
+public class CopyNumberCountFilter {
+
+    private String molecularProfileId;
```

Comment:
should support multi profile, use `profileType` instead of `molecularProfileId`

Reply:
Do you mean List<profileType>?